code/style.css

```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  background: white
 }
+
+
+header {
+  display: block;
+  height: 200px;
+  background-color: #ffffff;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}                                 
+
+
+.logo-img {
+  width: 250px;
+  margin: 0 0 0 2%;                    
+}
+
+
+nav {
+  align-self: flex-end;                    
+  margin: 14px;
+}
+
+
+.topnav {
+  background-color: #ffff;
+  overflow: hidden;
+  font-family: 'Bebas Neue', cursive;
+  
+}
+
+
+.topnav a {
+  float: left;
+  color: #c0c0c0;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 25px;
+}
+
+
+.topnav a:hover {
+  background-color: #c59000;
+  color: #393e46;
+}  
+
+
+article {
+  background: #FFF8F0; 
+  padding: 20px;
+}
+
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.big-news-img {
```

Comment:
The picture gets very big on large screens. A suggestion is to add a view height (vh) for the .big-news-img so the entire image and heading is shown when you enter the site (max-height: 55vh; object-fit:cover;) 
(maybe only add this for a media query).